bin/helpers/utils.js

```diff
@@ -745,3 +745,32 @@ exports.deleteBaseUrlFromError = (err) => {
   return err.replace(/To test ([\s\S]*)on BrowserStack/g, 'To test on BrowserStack');
 }
 
+exports.setBrowsers = async (bsConfig, args) => {
+  return new Promise((resolve, reject) => {
+    if(!this.isUndefined(args.browser)){
+      try{
+        bsConfig["browsers"] = []
+        let browsersList = args.browser.split(',')
+        browsersList.forEach((browser)=>{
```

Comment:
convert to simple for loop as forEach doesn't work with async await

Reply:
If there is an async-await inside forEach it doesn't work. By default, it does work fine.